pages/debsecan.md

```diff
@@ -0,0 +1,24 @@
+# debsecan
+
+> A tool to list vulnerabilities on a particular Debian installation. 
+> More information: https://gitlab.com/fweimer/debsecan.
+
+- List vulnerable installed packages on current host:
+
+`debsecan`
+
+- List vulnerable installed packages of a specific suite:
+
+`debsecan --suite {{release_code_name}}`
+
+- List only fixed vulnerabilities of testing ("sid"):
+
+`debsecan --suite sid --only-fixed`
+
+- List only fixed vulnerabilities of testing ("sid") and mail to root:
+
+`debsecan --suite sid --only-fixed --format report --mailto root --update-history`
+
+- Upgrade vulnerable installed packages:
+
+`sudo apt upgrade $(debsecan --only-fixed --format packages)`
```

Comment:
```suggestion
`sudo apt upgrade $(debsecan --only-fixed --format {{packages}})`
```

Reply:
Oh, actually the options of `--format` are `summary`(the default), `report`, `packages` etc (instead of *real {{package_name_like_debsecan_or_something_like_that}}*). Do I need to use brackets in every variant?